tron/bin/action_runner.py

```diff
@@ -93,6 +93,24 @@ def build_environment(run_id, original_env=None):
     return new_env
 
 
+def build_labels(run_id, original_labels=None):
```

Comment:
small suggestion:
```suggestion
def build_labels(
    run_id: str,
    original_labels: Optional[Dict[str, str]]=None,
) -> Dict[str, str]:
```
not 100% necessary since we're not running mypy on this file yet, but it'd be nice to add these in right now since we're pretty sure with the typing here :)

Reply:
added it now :)